geopandas/geoseries.py

```diff
@@ -1190,12 +1189,25 @@ def estimate_utm_crs(self, datum_name: str = "WGS 84"):
         """
         return self.values.estimate_utm_crs(datum_name)
 
-    def to_json(self, **kwargs) -> str:
+    def to_json(self, show_bbox=True, drop_id=False, to_wgs84=False, **kwargs) -> str:
```

Comment:
```suggestion
    def to_json(self, show_bbox:bool=True, drop_id:bool=False, to_wgs84:bool=False, **kwargs) -> str:
```
(geoseries.py is the only typing PR that has been merge so far, so it makes sense to keep types in the annotations where it's easy).

Reply:
Thanks Matt! I'll keep that in mind